student-store-express-api/app.js

```diff
@@ -1 +1,44 @@
-// YOUR CODE HERE
+const express = require('express')
+const cors = require('cors')
+
+const db = require('./data/db.json')
+const receiptCalculation = require('./receipt')
+
+const app = express()
+// const fs = require('fs')
+app.use(express.json())
+app.use(cors())
+
+app.get('/', (req,res) => {
+    res.status(200)
+    res.send(db)
+})
+
+app.get('/store/:id', (req,res) => {
+    const {id} = req.params
+
+    let myProduct = db.products.filter(product => product.id === parseInt(id))
+    myProduct ? myProduct = myProduct[0] : res.send ("No products found.")
+
+    res.status(200)
+    res.send({"product": myProduct})
```

Comment:
A more clear way to write these lines would be:

```
if (myProduct) {
  res.status(200).send({ "product": myProduct})
} else {
  res.status(200).send("No products found".)
}
```

This way you can keep the conditionals together. An additional change that could be helpful would be to also have the no products condition be the same data structure as when a product is found (an object).